addons/tmf_loadouts/loadouts/ussr_vdv_1980.hpp

```diff
@@ -0,0 +1,540 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
+// Reference: https://www.battleorder.org/vdv-footmobile-1986
+
+/*
+	Dependencies:
+		* CUP Units
+		* CUP Weapons
+*/
+
+//Created by AChesheireCat
+
+//Note: No body armor at ALL. Relatively light loadout because of that.
+
+class baseMan {// Weaponless baseclass
+	displayName = "Unarmed";
+	// All randomized.
+	uniform[] = {
+		"CUP_U_O_RUS_M88_VDV_rolled_up",
+		"CUP_U_O_RUS_M88_VDV"
+	};
+	vest[] = {
+		"CUP_V_RUS_Smersh_1",
+		"CUP_V_RUS_Smersh_2",
+		"CUP_V_O_SLA_Carrier_Belt03",
+		"CUP_V_O_Ins_Carrier_Rig"
+	};
+	backpack[] = {};
+	headgear[] = {
```

Comment:
Unsure about giving everyone booniehats.

Reply:
Yeah, not entirely sure either. However, most reference pictures that I found showed VDV guys running Afghankas, berets, and sometimes Ushankas.

Very rarely did they wear the penis helmets (lol).